src/index.js

```diff
@@ -25,7 +25,6 @@ function extend(base, props) {
 
 function setClass(attributes, value, append) {
 	let cl = getClass(attributes);
-	if (attributes.className) delete attributes.className;
```

Comment:
Not sure if this will mess up rendering.  Likely will render `class` twice (since its aliased)